openedx/adg/lms/applications/admin.py

```diff
@@ -230,9 +244,10 @@ def get_queryset(self, request):
 
     def applicant_name(self, obj):
         return obj.user.profile.name
+    applicant_name.short_description = _('Applicant Name')
```

Comment:
If a string is used more than once, that is susceptible to change, it is kept in constants.py. All other strings that are used only once are not placed in constants. You might find some exceptions in strings used in a single test twice/thrice, that are not placed in constants. That is because a string used in a test is not susceptible to change, like 'test_formset' . However, if there is a relatively generic test string that is used in multiple test cases, like 'Test note', then it is placed in constants.py. Placing every string in constants.py would be an overkill as it will result in a very long list of imports.